src/stubs.ts

```diff
@@ -324,6 +328,15 @@ export const createFigma = (config: TConfig): PluginAPI => {
       }
       this._textAutoResize = value;
     }
+    getRangeFontName(
+      start: number,
+      end: number
+    ): FontName | PluginAPI["mixed"] {
+      if (start && end && end >= start) {
+        return this._fontName || { family: "Roboto", style: "Regular" };
+      }
+      throw new Error(`Error: Invalid range: ${start}, ${end}`);
```

Comment:
`if (joinedConfig.simulateErrors)` is needed here

Reply:
Added 'joinedConfig.simulateErrors' with error handling for invalid start, end values.